File: C/exo4.c
#include <stdio.h>
#include <string.h>

int main(void)
{
    // ouvrir le fichier csv
    FILE* file = NULL;
    file = fopen("exo4.csv", "a");

    if(!file)
    {
        return 1;
    }
    // obtenir le modèle et la plaque
    char *modele = "206";
    char *plaque = "CO-VID-19";

    // imprimer dans le fichier
    fprintf(file, "%s, %s\n", modele, plaque);

    // fermer le fichier
    fclose(file);
}

File: Python/exo4.py
import csv

# on ouvre le fichier .csv
file = open("exo4.csv", "a")
# on indique ce que l'on veut écrire
modele = "206"
plaque = "CO-VID-19"

# ça écrit
writer = csv.writer(file)
writer.writerow((modele, plaque))

# on ferme le fichier .csv
file.close()